spot-ml/src/main/scala/org/apache/spot/netflow/FlowSuspiciousConnectsAnalysis.scala

```diff
@@ -44,8 +44,6 @@ object FlowSuspiciousConnectsAnalysis {
     val invalidFlowRecords = filterAndSelectInvalidFlowRecords(inputFlowRecords)
```

Comment:
it feels weird to me that logging of the corrupt/invalid records is happening after the analysis... shouldn't be happening in the front during the cleaning stages?